tests/integration_tests/utils.py

```diff
@@ -54,6 +54,9 @@ def driver():
             BOT_TOKEN="7arqwr6kzibc58zomct9ndfk1e",
             MATTERMOST_PORT=8065,
             SSL_VERIFY=False,
+            WEBHOOK_HOST_ENABLED=True,
+            WEBHOOK_HOST_URL="http://127.0.0.1",
+            WEBHOOK_HOST_PORT=8579
```

Comment:
The `driver` fixture is only used for sending messages and checking if a response was received, so it does not need a webhook server to be run (in fact I'm surprised this works, since we'll now be running two webhook servers on the same address and port?)

Reply:
Ah never mind, the `driver` bot is never actually started (we just obtain its driver) so the webhook settings aren't ever used anyway. I've removed them nonetheless, to prevent confusion